### ISSUE TITLE
Cannot send a delayed BYE to caller

### ISSUE THREAD

**Issue by oferbar** [SIPp/sipp#732]
Hello,

I am using SIPp v3.5.2-SCTP-PCAP-RTPSTREAM, with a custom configuration based on: uas.xml.
Normally, the caller is sending an INVITE and then a BYE to terminate the call.
However, for testing purposes I am trying to send a BYE from the server after X seconds.
I added an unused attribute in the INVITE header, and then parse it during the INVITE in the Xml.
If I get the header, a conditional <nop> is doing <pause> and <send BYE>.
However, that section does not work, although I verified that the condition is met with logs.

Bottom line, what is the best way to handle such a case inside SIPp?
So far, everything I tried did not work.

Here's some code snippets:

INVITE:
![image](https://github.com/SIPp/sipp/assets/1737075/f4c56c65-36aa-469e-8066-ff6097c1017a)

![image](https://github.com/SIPp/sipp/assets/1737075/d0e10c96-a5ac-4cc6-9e9e-c2e36df7f9e4)

Pause & BYE:

![image](https://github.com/SIPp/sipp/assets/1737075/7c7bb37c-d869-468c-b867-f3c6c3896148)

Thanks
